gateway/gateway_test.go

```diff
@@ -844,6 +844,9 @@ func TestProxyProtocol(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
+	EnablePort(p, "tcp")
```

Comment:
Should not it be "http" instead?

Reply:
nope, the test case is for `tcp` service.